litgpt/finetune/full.py

```diff
@@ -112,11 +119,27 @@ def main(
     out_dir: Path,
     train: TrainArgs,
     eval: EvalArgs,
+    longlora: LongLoraArgs,
 ) -> None:
     validate_args(train, eval)
+    if resume is True:
+        resume = max(out_dir.rglob("step-*/*.pth"), key=(lambda p: int(p.parent.name.split("-")[1])))
+    if resume:
```

Comment:
Those lines were in lines 142-143 before and I put those there because I need to resume the previous LongLoraArgs and overwrite the current ones. I think that you were doing that also before because `resume` can be both a boolean (and in that case the checkpoint file will be loaded from the folder with the greater step) and a path to a checkpoint file